apps/vgo-cli/src/vgo_cli/version_reminder.py

```
@@ -8,18 +8,20 @@
-from vgo_cli.upgrade_service import refresh_installed_status, refresh_upstream_status
+from vgo_cli.upgrade_service import has_recorded_install_truth, load_recorded_install_status, refresh_upstream_status
 from vgo_cli.upgrade_state import is_upstream_cache_stale
 
 
 def build_update_reminder(repo_root: Path, target_root: Path, host_id: str) -> str | None:
     try:
-        status = refresh_installed_status(repo_root, target_root, host_id)
+        status = load_recorded_install_status(repo_root, target_root, host_id)
         if is_upstream_cache_stale(status):
             status = refresh_upstream_status(repo_root, target_root, status)
     except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)